src/cython_cmake/cmake/FindCython.cmake

```diff
@@ -13,6 +13,13 @@
 #  ``CYTHON_FOUND``
 #    true if the program was found
 #
+# And the following target:
+#
+#  ``Cython::Cython``
+#    The Cython executable
```

Comment:
Should we stop documenting the `CYTHON_EXECUTABLE` variable altogether and just say that the only supported way to get it is via this target?

Reply:
They are different when cross compiling. Targets will use the cross-compiling emulator, while variables are just variables. Also, target's can't be used in some places like during the configure process, while variables can.

Reply:
Good point, I guess the cython command could itself have been cythonized and therefore be native to the target system.